### DISCUSSION TITLE
Action 559517只能下載最新的33-22章節

### DISCUSSION THREAD

**Discussion by 123hi123** [hect0x7/JMComic-Crawler-Python#253] [Q&A] [answered]
大概看過大家提的issue和discuss 看起來是option裡面可以調整這部分，但我希望可以在action使用下載功能所以不太確定怎麼修改才能一次下載全部章節，我會繼續研究，有知道的大佬也可以在這裡造福大家

**Comment by hect0x7** [comment, marked as answer]
应该是因为这个本的部分章节需要登录才能观看。你在option里配置下登录插件试试。
